tests/assert.go

```diff
@@ -0,0 +1,16 @@
+package tests
+
+import "testing"
+
+func AssertPanic(t *testing.T, f func(), expectedError string) {
+	t.Helper()
+
+	defer func() {
+		err := recover()
+		if err != expectedError {
+			t.Errorf("The code did not panic with expected error. Got: %s", err)
+		}
+	}()
+
```

Comment:
We have testify in dependencies, you can panic assertion from this library https://pkg.go.dev/github.com/stretchr/testify/assert#PanicsWithError